services/api/src/messaging/players.service.ts

```diff
@@ -54,7 +82,18 @@ const publishFinalBoard = (ws: GameWebSocket): void => {
   });
 };
 
+const publishGameState = (ws: GameWebSocket): void => {
+  publish(ws.roomId, {
+    type: MessageType.UpdateGameState,
+    payload: gameRoomService.getGameState(ws.roomId),
+  });
+};
+
 export const figureMoved: Handler = (ws, payload: PlaceFigureMessage): void => {
+  if (gameRoomService.getGameState(ws.roomId) !== GameState.GAME_IN_PROGRESS) {
```

Comment:
this seems like some sort of workaround. Why is this needed?

Reply:
Its just for safety reasons. Even if the game state would be GameNotStarted and user could not place figure on the board, we could send a message with a bot, and probably crash the game.

Reply:
Its for a safety reasons. Without this check, even if game state is `GameNotStarted` and user cannot place figure on the board, it would be possible to send a message for example with a bot